src/main/java/sungshin/sooon/controller/PostController.java

```diff
@@ -0,0 +1,95 @@
+package sungshin.sooon.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import sungshin.sooon.dto.*;
+import sungshin.sooon.model.Account;
+import sungshin.sooon.model.CurrentUser;
+import sungshin.sooon.model.Post;
+import sungshin.sooon.repository.PostRepository;
+import sungshin.sooon.service.PostService;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/post")
```

Comment:
3. 복수형 명사를 사용하라
단수형과 복수형을 섞어서 사용하지마라. 간단하게 유지하고 모든 자원에 오직 복수형 명사만 사용하라.

출처: https://multifrontgarden.tistory.com/252 [우리집앞마당]
```suggestion
@RequestMapping("/api/v1/posts")
```